README.md

```diff
@@ -14,7 +14,19 @@ Firebolt GO driver is an implementation of `database/sql/driver`.
 go get github.com/firebolt-db/firebolt-go-sdk
 ```
 
-### Example
+### DSN (Data source name)
+All information for the connection should be specified using the DSN string. The firebolt dsn string has the following format:
+```
+firebolt://[/database]?account_name=account_name&client_id=client_id&client_secret=client_secret[&engine=engine]
+```
+
+- **client_id** - credentials client id.
+- **client_secret** - credentials client secret.
+- **account_name** - the name of Firebolt account to log in to.
+- **database** - (optional) the name of the database to connect to.
+- **engine** - (optional) the name of the engine to run SQL on.
+
```

Comment:
Let's stress that client id and secret are coming from service account and link the docs. We've already had someone stumbling on it.